src/app/domains/Question/components/QuestionForm/QuestionForm.js

```diff
@@ -50,116 +91,80 @@ function QuestionForm(props) {
                   content={<QuestionTypeSelect />}
                 />
               </Col>
             </Row>
-            <Row noGutters>
-              <Col>
-                <QuestionHeader
-                  titlePlaceholder={'Question two'}
-                  subtitlePlaceholder={'Question with TextArea'}
-                />
-              </Col>
-            </Row>
-            <Row noGutters>
-              <Col>
-                <TextAreaForm
-                  noBorder
-                  btnProps={{ children: 'Submit', type: 'primary' }}
-                />
-              </Col>
-            </Row>
-          </Card>
-        </Col>
-      </Row>
-
-      <Row noGutters mb={2}>
-        <Col>
-          <Card style={styles.cardStyle}>
             <Row noGutters h="between">
               <Col cw="auto">
-                <Tag color="blue">Question 3</Tag>
-              </Col>
-
-              <Col cw="auto">
-                <Popover
-                  trigger={'click'}
-                  placement={'bottomRight'}
-                  btnType="primary"
-                  btnIcon={<SettingOutlined />}
-                  content={<QuestionTypeSelect />}
-                />
-              </Col>
-            </Row>
-            <Row noGutters>
-              <Col>
                 <QuestionHeader
-                  titlePlaceholder={'Question three'}
-                  subtitlePlaceholder={'Question with Rating'}
+                  titlePlaceholder={'change on Question type'}
+                  subtitlePlaceholder={'Description(optional)'}
                 />
               </Col>
             </Row>
+            {question.layoutType === LAYOUT_TYPE.BETWEEN && (
+              <Row>
+                <Col cw="auto">
+                  <Box
+                    {...question.layoutType}
+                    backgroundImage={`url(https://helpx.adobe.com/content/dam/help/en/photoshop/using/convert-color-image-black-white/jcr_content/main-pars/before_and_after/image-before/Landscape-Color.jpg)`}
+                  />
+                </Col>
+              </Row>
+            )}
             <Row noGutters>
               <Col>
-                <Rate />
+                {cloneElement(
+                  questionTypesMap[question.questionType].component,
+                  {
+                    ...question
+                  }
+                )}
               </Col>
             </Row>
           </Card>
         </Col>
+        {layoutSides.includes(question.layoutType) && (
+          <Col
+            v="center"
+            display="flex"
+            style={styles.columnStyle}
+            height="100%"
+            order={rightSide.includes(question.layoutType) ? 3 : 1}>
+            <Box
+              {...question.layoutType}
+              backgroundImage={`url(https://helpx.adobe.com/content/dam/help/en/photoshop/using/convert-color-image-black-white/jcr_content/main-pars/before_and_after/image-before/Landscape-Color.jpg)`}
```

Comment:
Same here, please, move url to constant variable

Reply:
Already answered this question above